app.go2tv.go2tv.yml

```diff
@@ -0,0 +1,39 @@
+app-id: app.go2tv.go2tv
+runtime: org.freedesktop.Platform
+runtime-version: &runtime-version "24.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    version: *runtime-version
+
+command: go2tv
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --device=dri
+  - --filesystem=host
```

Comment:
why is host needed?

Reply:
The app streams media files to various devices, either through the built-in file browser or by drag-and-drop. Since files can come from anywhere on the system, filesystem=host is essential. Without access to the host filesystem, the app cannot function